### DISCUSSION TITLE
Disabling Topic Configurations

### DISCUSSION THREAD

**Discussion by guzelcihad** [tchiotludo/akhq#1218] [Q&A] [answered]
Hi,

As you can see in the below picture, a user which has reader role can edit this configuration. The applications doesn't restrict to edit this html inputs. I've opened the browser developer tools and saw that when a user changes these form element, there is no request goes to application. This is fine but it creates a feeling that a user can change this settings. Any way to prevent that?

![Screen Shot 2022-10-03 at 12 59 48](https://user-images.githubusercontent.com/18442727/193551543-0cc19183-e3c6-4958-b624-40ba3e717cf6.png)

**Comment by tchiotludo** [comment, marked as answer]
It's a bug, I've created the issue [here](https://github.com/tchiotludo/akhq/issues/1219)
